test/e2e/sync/sync.go

```diff
@@ -232,14 +232,15 @@ func createObjects() error {
 // Delete Temporarily created local object files and spec file
 func deleteTempFiles() error {
 	klog.Infoln("STEP: Delete created Files")
-	specFolder := strings.Split(SpecFileName, "/")[0]
-	klog.Infoln("Spec folder : ", specFolder)
+	specFolder := strings.Split(SpecFileName, "/")[1]
```

Comment:
lets explore if we can use the `filepath.Clean()` function

Reply:
`filepath.Clean(SpecFileName)` returns `syncXXXXXXX/files` from `./syncXXXXXXX/files` which further required retrieving the directory name, leading to two-step approach. 

Alternatively, `filepath.Dir(SpecFileName)` returns the required directory name alone `syncXXXXXXX` which needs to be removed with all its content. The same has been adopted.